guide-projects/spring-boot-eventeria-guide/src/main/resources/application.yml

```diff
@@ -0,0 +1,59 @@
+spring:
+  cloud:
+    function:
+      definition:
+        transformCloudEventToMessage|toNotifyToSubscribers;
+        transformMessageToCloudEvent;
```

Comment:
Is this property also necessary?

Reply:
No. This function is invalid because functional publisher does not work